manager/src/main/java/io/gingersnapproject/Caches.java

```diff
@@ -82,9 +87,17 @@ private void actualRemove(String name, String key, Uni<String> value) {
       }
    }
 
-   public void put(String name, String key, String value) {
+   public Uni<String> put(String name, String key, String value) {
+      Uni<String> indexUni = searchBackend.put(name, key, value)
+                  .map(___ -> value)
+            // Make sure subsequent subscriptions don't update index again
+                  .memoize().indefinitely();
```

Comment:
We'll end up indexing db-syncer offset and schema.

Reply:
This is handled by https://github.com/gingersnap-project/cache-manager/pull/21